test/api/sandbox_test.dart

```diff
@@ -0,0 +1,110 @@
+import 'dart:typed_data';
+
+import 'package:arweave/src/api/sandbox.dart' as sandbox;
+import 'package:test/test.dart';
+
+import 'sandbox_test_utils.dart' as sandbox_test_utils;
+
+/// TODO: update the test cases
+// typedef SandboxDomain = String;
+// typedef TxId = String;
+// final Map<SandboxDomain, TxId> testBaseline = {
+//   'f3valm4d36fd6w4uhuibvjrnwra3entolqiojhlcsghhnp32pstq':
+//       'LuoFs4Pfij9blD0QGqYttEGyNm5cEOSdYpGOdr96fKc',
+//   'flqlektwzxrgtlpmce53dnnej2clq5njgbbh2tztjbs4aqz3owjq':
+//       'KuCyKnbN4mmt7BE7sbWkToS4dakwQn1PM0hlwEM7dZM',
+//   '6c6fodlctydufzmbsmzykkweigtk5w5zrqauiyetmcaqukgm':
+//       '8LxXDWKeB_0LlgZMzhSrEQaau27-mMAURgk2CBCijMY',
+//   'oerew5mrlsda5yrsmml3vwpxb6tvpobbzyfb4l5pucthyyhmbe':
+//       'cSJLdZFchg7iMmMXutn3D6dXuCHOCh4_vr6CmfGDsCc',
+//   'vxxdwy6scapybz7ekkw32ua5naseioagnu4uutbq3cftvivl':
+//       're47Y9IQH4-Dn5FKtvVAdaCRE-OAZtOUpMMNiLOqKrw',
+//   'bujjxactpuy7cmm3ul6fjuwuqnbymloddqvsjswhxsvkd255bq':
+//       'DRKbgFN9MfExm6L8VNLUg0OGLcMcKyTKx7yq_oeu9DM',
+//   'b4zy2sx2upeczfaxpscxfuglgh2mljo4gu6ycuhujgz5yjbc':
+//       'DzONSvqjyCyUF3yFctDLMfTFpd_w-1PYFQ9Emz3CQic',
+//   'kruhyqr3j24gxknxda2pbqa6ru2ctocctwr4z3hesnjukreb':
+//       'VGh8QjtOuGuptxg08MAejTQpuEKdo8zs_5JNTRUSB_Q',
+// };
```

Comment:
Multiple test cases vs one single case?

Reply:
you can keep multiple as you have them already, but they will not bring too much value

Reply:
I simply took the TX IDs already present in there to use as input for the method.
29bfd003bca6dee8083ce6fab945f9c4edc0f8f1